testing/vcs/test_script_for_py.py

```diff
@@ -0,0 +1,15 @@
+import vcs
+
+list=["boxfill","meshfill","isofill","textcombined","texttable","vector"]
```

Comment:
You shouldn't name things reserved words in python (`list` being the type of `[]`). It technically works but is bad form, and will break things later down the road.